src/lib/toolchain_test.rs

```diff
@@ -124,16 +124,19 @@ fn get_cargo_binary_path_valid() {
 #[test]
 #[ignore]
```

Comment:
This test is ignored in CI. Any reason for this @sagiegurari

Reply:
hmm. I'm not entierly sure how `ci-flow` executes this particular test, but my guess is that since `env::set_var` isn't safe for multi-threaded environments and some other test executing in parallel will change the env var and that spuriously causes this test to fail.